### PR TITLE
feat: Add imagebuilder in DevOps module

### PR DESCRIPTION
**Pull request by Ganbingkun**
### What type of PR is this?
/kind design

### What this PR does / why we need it:

Add a Shipwright-based ImageBuilder under the devpos page

```release-note
None
```

### CODE REVIEW

File: src/pages/devops/containers/ImageBuilder/Detail/Environment/index.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    if (this.module === 'containers') return [data]
+
+    if (!isEmpty(containers)) return containers
+    if (!isEmpty(spec)) return get(spec, 'template.spec.containers', [])
+
+    return []
+  }
+
+  get test() {
```
Comment (harrisonliu5) [inline src/pages/devops/containers/ImageBuilder/Detail/Environment/index.jsx:68]: should remove it

**Comment by harrisonliu5** [comment]
You need to make improvements to the following:
1. Do not use Chinese comments in the code.
2. Fix the regex that the CodeQL prompt
3. Modify PR following [Conventional Commits 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/)

We hope to hear from you as soon as possible, and thanks for your contribution.

File: src/pages/devops/containers/ImageBuilder/Detail/Environment/index.jsx (hunk 1 above)
Comment (Ganbingkun) [reply src/pages/devops/containers/ImageBuilder/Detail/Environment/index.jsx]: ok

**Pull request merged**